### ISSUE TITLE
cat not find the file of ui-overview.md

### ISSUE THREAD

**Issue by coki230** [apache/incubator-skywalking#2137]
Please answer these questions before submitting your issue.

- Why do you submit this issue?
- [ ] Bug

___
### Bug
document bug

___
### Requirement or improvement
it cat not find the file of ui-overview.md in
- If you want to customize UI, start with [UI overview](ui-overview.md) document.

**Comment by wu-sheng** [comment]
Yes, this ia a missing document. It should be easy, but I forget to write it, thanks to tip.
